src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -25,76 +25,87 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
-    };
-  </script>
+<title><%=conversation.getTitle()%></title>
+<link rel="stylesheet" href="/css/main.css" type="text/css">
+
+<style>
+#chat {
+	background-color: white;
+	height: 500px;
+	overflow-y: scroll
+}
+</style>
+
+<script>
+	// scroll the chat div to the bottom
+	function scrollChat() {
+		var chatDiv = document.getElementById('chat');
+		chatDiv.scrollTop = chatDiv.scrollHeight;
+	};
+</script>
 </head>
 <body onload="scrollChat()">
 
-  <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
-    <a href="/conversations">Conversations</a>
-      <% if (request.getSession().getAttribute("user") != null) { %>
-    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else { %>
-      <a href="/login">Login</a>
-      <a href="/register">Register</a>
-    <% } %>
-    <a href="/about.jsp">About</a>
-  </nav>
-
-  <div id="container">
-
-    <h1><%= conversation.getTitle() %>
-      <a href="" style="float: right">&#8635;</a></h1>
-
-    <hr/>
-
-    <div id="chat">
-      <ul>
-    <%
-      for (Message message : messages) {
-        String author = UserStore.getInstance()
-          .getUser(message.getAuthorId()).getName();
-    %>
-
-      <li><a href="/profile/<%=author%>"><%= author %></a>: <%= message.getContent() %></li>
-    <%
-      }
-    %>
-      </ul>
-    </div>
-
-    <hr/>
-
-    <% if (request.getSession().getAttribute("user") != null) { %>
-    <form action="/chat/<%= conversation.getTitle() %>" method="POST">
-        <input type="text" name="message">
-        <br/>
-        <button type="submit">Send</button>
-    </form>
-    <% } else { %>
-      <p><a href="/login">Login</a> to send a message.</p>
-    <% } %>
-
-    <hr/>
-
-  </div>
+	<nav>
+		<a id="navTitle" href="/">TeamRocket Chat App</a> <a href="/conversations">Conversations</a>
+		<%
+		if (request.getSession().getAttribute("user") != null) {
+			%>
+			<a href="/feed">Activity Feed</a>
+			<a>Hello <%=request.getSession().getAttribute("user")%>!</a>
+			<%
+		} else {
+			%>
+			<a href="/login">Login</a> <a href="/register">Register</a>
+			<%
+		}
+		%>
+		<a href="/about.jsp">About</a>
+	</nav>
+
+	<div id="container">
+
+		<h1><%=conversation.getTitle()%>
+			<a href="" style="float: right">&#8635;</a>
+		</h1>
+
+		<hr/>
+
+		<div id="chat">
+			<ul>
+				<%
+				for (Message message : messages) {
+					String author = UserStore.getInstance().getUser(message.getAuthorId()).getName();
+				%>
+					<li><strong><%=author%>:</strong> <%=message.getContent()%></li>
+				<%
+				}
+				%>
+			</ul>
+		</div>
+
+		<hr />
+
+		<%
+		if (request.getSession().getAttribute("user") != null) {
+		%>
+			<form action="/chat/<%=conversation.getTitle()%>" method="POST">
+				<input type="text" name="message"> <br />
+				<button type="submit">Send</button>
+			</form>
+		<%
+		} else {
+		%>
+			<p>
+				<a href="/login">Login</a> to send a message.
+			</p>
+		<%
+		}
+		%>
+
+		<hr/>
+
+	</div>
```

Comment:
I don't understand why this file was changed other than adding the link to the feed. It seems like the indentation is the only thing that was changed and it was more correct before.